packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -6,6 +6,70 @@ import { Construct } from 'constructs';
 import { CfnJobDefinition } from './batch.generated';
 import { JobDefinitionImageConfig } from './job-definition-image-config';
 
+/**
+ * The log driver to use for the container.
+ */
+export enum LogDriver {
```

Comment:
Any reason you didn't add `logentries`? 

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-batch-jobdefinition-containerproperties-logconfiguration.html#cfn-batch-jobdefinition-containerproperties-logconfiguration-logdriver

> The supported log drivers are awslogs, fluentd, gelf, json-file, journald, logentries, syslog, and splunk.

Reply:
I did not add `logentries` because there was no description for the same in the section for `LogDriver`. I only included the ones that had a description. I can add this one as well if you think it should be fine.

Reply:
Added in next commit.